README.md

```diff
@@ -5,39 +5,58 @@
 ![Screenshot](https://user-images.githubusercontent.com/910672/45590262-55333c00-b90b-11e8-9f95-c360d35ced5f.jpg)
 
 ## What is this?
+
 An open source scrobbling client for the web. Just login and scrobble the track(s) you want! This tool lets you scrobble songs you heard on the radio, vinyls or at some place where you need to manually scrobble.
 
 You may use it directly from here: [https://openscrobbler.com/](https://openscrobbler.com/)
 
 What is _scrobbling_, you ask? It's tracking the music you listen to, using [last.fm](https://last.fm/).
 
 ## Help us make it better!
+
 This tool is open source and free. You can support its development by [becoming a Patron](https://www.patreon.com/OpenScrobbler) (you'll have a distinct identifier in the application and get a sneak peek into what's coming next! :sparkles: )
 
 ### Contributing
+
 Contributions and suggestions are welcome. Please read the [contribution guidelines](https://github.com/elamperti/OpenWebScrobbler/blob/main/CONTRIBUTING.md) for more information. Issues labeled with ["help wanted"](https://github.com/elamperti/OpenWebScrobbler/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22) are great candidates if you don't know where to begin!
 
 ### Translations
+
 To join the translators team, follow the link at the bottom of the language selector in the application ("Want to add your language?").
 
 ## Development setup
-You'll need [a Last.fm API account](https://www.last.fm/api/account/create)! (and probably a Discogs API key as well)
+
+You need a [Last.fm API account](https://www.last.fm/api/account/create) to be able to interact with OpenWebScrobbler and query Last.fm database.
+
+You need a [Discogs API application](https://www.discogs.com/settings/developers) to be able to query Discogs database.
```

Comment:
Possible simplification/improvement for these lines and the list below:

```suggestion

  - Clone the repository
  - Copy `.env` to `.env.development.local`, then set at least  the required variables
    - You'll need a [Last.fm API account](https://www.last.fm/api/account/create) to be able to interact with Open Scrobbler (it's used for authentication and queries). Once you have your keys, fill in `REACT_APP_LASTFM_API_KEY` and `LASTFM_API_KEY` (same value in both) and `LASTFM_SECRET`

    - Optional:  to interact with Discogs, create a [Discogs application](https://www.discogs.com/settings/developers) to get API keys and fill in `DISCOGS_API_KEY` and `DISCOGS_SECRET`
```

I'd also add two comments in `.env` specifying required/optional sections:

```
# Required
REACT_APP_HOST=localhost:3000
REACT_APP_LASTFM_API_KEY=
LASTFM_API_KEY=
LASTFM_SECRET=
SQLITE_DB_PATH=/var/www/html/database/development.db

# Optional
DISCOGS_API_KEY=
DISCOGS_SECRET=
REACT_APP_ANALYTICS_MEASUREMENT_ID=
REACT_APP_GROWTHBOOK_API_KEY=
REACT_APP_LINK_TO_TRANSLATIONS="https://github.com/elamperti/OpenWebScrobbler/blob/main/CONTRIBUTING.md#translations"
REACT_APP_SENTRY_DSN=

# Only for development: uncomment and set these numbers to your user/group IDs
# APACHE_RUN_USER=#1000
# APACHE_RUN_GROUP=#1000
```